### ISSUE TITLE
Verify Command: Version checking of required packages: node, docker and docker-compose

### ISSUE THREAD

**Issue by realgarp** [nemtech/symbol-bootstrap#119]
When setting up a new Symbol testnet node I ran into a version issue of docker-compose.
The default docker-compose on my Ubuntu 18.04 LTS was apparently not the version symbol-bootstrap expected, because docker-compose version complained about the format of the yml file generated by symbol-bootstrap. Trying to start the node with symbol-bootstrap does not work and exits in that case.
So I had to manually upgrade docker-compose to 1.27.4, to solve the issue.

The symbol-bootstrap docs mentions three required packages: node, docker and docker-compose.
It would be an improvement if symbol-bootstrap would be able to 
- check the installed versions of those three packages
- provide the docker and docker-compose versions also in the output of the -v command, as is already done for node.
- and if the required versions are not present: be able to indicate that to the user and not run at all before the user fixes it by upgrading the relevant package(s)

Thanks for considering... it would help future node owners.

**Comment by segfaultxavi** [comment]
@fboucquez If you tell me the required versions for `docker` and `docker-compose` I'll add the info to the docs.

**Comment by segfaultxavi** [comment]
I think I ran into the same issue on my Ubuntu 18.04, my `docker-compose` 1.25 crashed with:
```
AttributeError: module 'six' has no attribute 'ensure_text'
```
Is this the same thing you saw, @realgarp ?

**Comment by segfaultxavi** [comment]
From a new folder:
```
$ docker --version
Docker version 19.03.8, build afacb8b7f0

$docker-compose --version
docker-compose version 1.25.0, build unknown

$ symbol-bootstrap --version
symbol-bootstrap/0.4.0 linux-x64 node-v15.0.1

$ symbol-bootstrap start -p bootstrap -a full -t test
                            _               _           _                       _           _                          
  ___   _   _   _ __ ___   | |__     ___   | |         | |__     ___     ___   | |_   ___  | |_   _ __    __ _   _ __  
 / __| | | | | | '_ ` _ \  | '_ \   / _ \  | |  _____  | '_ \   / _ \   / _ \  | __| / __| | __| | '__|  / _` | | '_ \ 
 \__ \ | |_| | | | | | | | | |_) | | (_) | | | |_____| | |_) | | (_) | | (_) | | |_  \__ \ | |_  | |    | (_| | | |_) |
 |___/  \__, | |_| |_| |_| |_.__/   \___/  |_|         |_.__/   \___/   \___/   \__| |___/  \__| |_|     \__,_| | .__/ 
        |___/                                                                                                   |_|    
2021-01-18T10:44:57.046Z info     Generating config from preset bootstrap
2021-01-18T10:44:57.048Z info     Assembly preset full
2021-01-18T10:44:57.292Z info     User for docker resolved: 1000:1000
2021-01-18T10:44:57.294Z info     Running image using Exec: symbolplatform/symbol-server:tools-gcc-0.10.0.5 bash createCertificate.sh
2021-01-18T10:44:57.305Z info     Running image using Exec: symbolplatform/symbol-server:tools-gcc-0.10.0.5 bash createCertificate.sh
2021-01-18T10:44:57.315Z info     Running image using Exec: symbolplatform/symbol-server:tools-gcc-0.10.0.5 bash createCertificate.sh
2021-01-18T10:44:58.124Z info     Certificate for node peer-node-0 created
2021-01-18T10:44:58.154Z info     Certificate for node api-node-0 created
2021-01-18T10:44:58.283Z info     Certificate for node peer-node-1 created
2021-01-18T10:44:58.425Z info     Non-voting node api-node-0.
2021-01-18T10:44:58.426Z info     Running image using Exec: symbolplatform/symbol-server:tools-gcc-0.10.0.5 /usr/catapult/bin/catapult.tools.votingkey --secret=HIDDEN_KEY --startEpoch=1 --endEpoch=26280 --output=/votingKeys/private_key_tree1.dat
2021-01-18T10:44:58.432Z info     Running image using Exec: symbolplatform/symbol-server:tools-gcc-0.10.0.5 /usr/catapult/bin/catapult.tools.votingkey --secret=HIDDEN_KEY --startEpoch=1 --endEpoch=26280 --output=/votingKeys/private_key_tree1.dat
2021-01-18T10:45:03.171Z info     Voting key executed for node peer-node-0!
2021-01-18T10:45:03.321Z info     Voting key executed for node peer-node-1!
2021-01-18T10:45:03.466Z info     Voting Key Link Transaction Short Key V1 resolved
2021-01-18T10:45:03.479Z info     Voting Key Link Transaction Short Key V1 resolved
2021-01-18T10:45:03.506Z info     Running image using Exec: symbolplatform/symbol-server:tools-gcc-0.10.0.5 /usr/catapult/bin/catapult.tools.nemgen --resources=/userconfig --nemesisProperties=./userconfig/block-properties-file.properties
2021-01-18T10:45:04.192Z info     Nemgen executed!!!!
2021-01-18T10:45:04.221Z info     Configuration generated.
2021-01-18T10:45:04.248Z info     creating docker-compose.yml from last used profile.
2021-01-18T10:45:04.256Z info     docker-compose.yml file created /home/xartigas/networks/test/docker/docker-compose.yml
2021-01-18T10:45:04.271Z info     Spawn command: docker-compose -f test/docker/docker-compose.yml up --remove-orphans
Traceback (most recent call last):
  File "/usr/bin/docker-compose", line 11, in <module>
    load_entry_point('docker-compose==1.25.0', 'console_scripts', 'docker-compose')()
  File "/usr/lib/python3/dist-packages/compose/cli/main.py", line 72, in main
    command()
  File "/usr/lib/python3/dist-packages/compose/cli/main.py", line 125, in perform_command
    project = project_from_options('.', options)
  File "/usr/lib/python3/dist-packages/compose/cli/command.py", line 53, in project_from_options
    return get_project(
  File "/usr/lib/python3/dist-packages/compose/cli/command.py", line 151, in get_project
    return Project.from_config(
  File "/usr/lib/python3/dist-packages/compose/project.py", line 102, in from_config
    service_networks = get_networks(service_dict, networks)
  File "/usr/lib/python3/dist-packages/compose/network.py", line 320, in get_networks
    networks[network.true_name] = netdef
  File "/usr/lib/python3/dist-packages/compose/network.py", line 125, in true_name
    self._set_legacy_flag()
  File "/usr/lib/python3/dist-packages/compose/network.py", line 146, in _set_legacy_flag
    data = self.inspect(legacy=True)
  File "/usr/lib/python3/dist-packages/compose/network.py", line 106, in inspect
    return self.client.inspect_network(self.legacy_full_name)
  File "/usr/lib/python3/dist-packages/docker/utils/decorators.py", line 19, in wrapped
    return f(self, resource_id, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/docker/api/network.py", line 212, in inspect_network
    res = self._get(url, params=params)
  File "/usr/lib/python3/dist-packages/docker/utils/decorators.py", line 46, in inner
    return f(self, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/docker/api/client.py", line 230, in _get
    return self.get(url, **self._set_request_timeout(kwargs))
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 546, in get
    return self.request('GET', url, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 519, in request
    prep = self.prepare_request(req)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 452, in prepare_request
    p.prepare(
  File "/usr/lib/python3/dist-packages/requests/models.py", line 313, in prepare
    self.prepare_url(url, params)
  File "/usr/lib/python3/dist-packages/requests/models.py", line 379, in prepare_url
    scheme, auth, host, port, path, query, fragment = parse_url(url)
  File "/usr/lib/python3/dist-packages/urllib3/util/url.py", line 407, in parse_url
    ensure_func = six.ensure_text
AttributeError: module 'six' has no attribute 'ensure_text'
Process exited with code 1 and signal null
Process closed with code 1
```

**Comment by realgarp** [comment]
> 
> 
> I think I ran into the same issue on my Ubuntu 18.04, my `docker-compose` 1.25 crashed with:
> 
> ```
> AttributeError: module 'six' has no attribute 'ensure_text'
> ```
> 
> Is this the same thing you saw, @realgarp ?

In my case the older version of docker-compose complained about the added "version" line in the yml file

**Comment by fboucquez** [comment]
Yes, Erik error is the version problem, fixed with docker-compose version 1.25.0.

Unsure about Xavi's error. The stack trace is from docker compose's python. I haven't seen it before.

**Comment by fboucquez** [comment]
Verify command added to 1.0.4
